pycloudlib/gce/util.py

```diff
@@ -0,0 +1,19 @@
+"""Common GCE utils."""
+
+
+def raise_on_error(response):
+    """Look for errors in response and raise if found."""
+    if 'httpErrorStatusCode' in response:
+        raise Exception(
```

Comment:
Is it worth fielding urllib.error.HTTPError? maybe raise RuntimeError to make the error a bit more specialized than Exception?

Reply:
Sure, I can make a custom exception for the other one too